src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -167,26 +180,92 @@ define(function (require, exports, module) {
             var sectionClasses = classnames({
                 "libraries": true,
                 "section": true,
-                "section__collapsed": !this.props.visible
+                "section__collapsed": !this.props.visible,
+                "libraries_no-drop": this.state.isDropTarget && !this.state.selectedLibrary
```

Comment:
`selectedLibrary` will always be null when no connection, so it should be enough of checking here.